src/selectors/metadata.js

```diff
@@ -56,6 +58,12 @@ export const getClickedNodeMetaData = createSelector(
     if (!node) {
       return null;
     }
+    //rounding of metrics data
+    nodeMetrics[node.id] &&
+      Object.entries(nodeMetrics[node.id]).forEach(([key, value]) => {
+        nodeMetrics[node.id][key] = Math.round(value * 100) / 100;
```

Comment:
I see that we are rounding this up to 2 decimal values - for displaying rounded up numbers in our codebase we usually use `toFixed`  rather than using `Math.round`  ( which in our case, would be `+value.toFixed(2)` to eliminate the unnecessary zeros).

In terms of accuracy, the results of both are very similar ( both `toFixed` and `Math.round` would still throw oddities for certain edge cases), however `toFixed(2)` is a much more readable notation.

Reply:
@studioswong  - toFixed() returns a string and it messes up the way the data is represented on the json tree and can confuse the user hence it was decided to use Math.round() instead.